modules/fundamental/src/advisory/endpoints/test.rs

```diff
@@ -612,3 +615,50 @@ async fn delete_advisory(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+#[test_context(TrustifyContext)]
+#[test(actix_web::test)]
+async fn query_advisories_by_label(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    let query = async |q| {
+        let app = caller(ctx).await.unwrap();
+        let uri = format!("/api/v2/advisory?q={}", encode(q));
+        let req = TestRequest::get().uri(&uri).to_request();
+        let response: Value = app.call_and_read_body_json(req).await;
+        assert_eq!(1, response["total"], "for {q}");
+    };
+    ctx.ingest_document_as(
+        DOC,
+        Format::CSAF,
+        [
+            ("type", "csaf"),
+            ("source", "test"),
+            ("importer", "none"),
+            ("file", "cve-2023-33201.json"),
+            ("datasetFile", "none"),
+            ("foo", "bar"),
+        ],
+    )
+    .await?;
+
+    query("labels:type!=spdx").await;
```

Comment:
Just looking what others do. GitHub is using the singular, which also makes more sense to me:

![image](https://github.com/user-attachments/assets/cebedb29-ca14-42eb-b1f2-863dd2bf49a3)

Reply:
Hm. I also notices some other examples using the singular. Feels a bit confusing to me. Is both allowed?

Reply:
[yes](https://github.com/trustification/trustify/blob/7bcb9939d3387b99e7d364dd0f5285f143623df4/modules/fundamental/src/sbom/service/sbom.rs#L140)